go.mod

```diff
@@ -30,6 +30,7 @@ require (
 	github.com/jaegertracing/jaeger v1.22.0
 	github.com/leoluk/perflib_exporter v0.1.0
 	github.com/mattn/go-colorable v0.1.7 // indirect
+	github.com/mostynb/go-grpc-compression v1.1.6
```

Comment:
This repository is not very active/used. Are we ok to depend on it?

Reply:
Good point, @lizthegrey WDYT?

Reply:
The lz4 code from this repository seems simple enough to be duplicated here, with the appropriate credits...

Reply:
seems to get updated with klauspost/compress which is good enough for me.